cypress/e2e/dev.spec.ts

```diff
@@ -0,0 +1,17 @@
+import '../support/commands';
+
+describe('example test', () => {
```

Comment:
nit: "example test" - copy paste error?

Reply:
No, this was intentional... just a demonstration of how to login/logout using the added Cypress command.

Do you think we need tests to actually test the setup of the custom commands? 🤔

Reply:
Haha fair enough! Probably not then